Resources/Prototypes/_LostParadise/Maps/Hell/H_kerberos.yml

```
@@ -0,0 +1,77 @@
+            SalvageSpecialist: [ 3, 3 ]
+            # Civilian
+            Passenger: [ -1, -1 ]
+            Clown: [ 1, 1 ]
+            Mime: [ 1, 1 ]
+            Musician: [ 1, 3 ]
+            Reporter: [ 1, 2 ]
+            # Justice
+            Prosecutor: [ 1, 1 ]
+            Clerk: [ 2, 2 ]
+            ChiefJustice: [ 1, 1 ]
+            #LOST PARADISE
```

yamllint

[warning] missing starting space in comment

(comments)